src/KeyVault/KeyVault.sln

```diff
@@ -55,14 +55,14 @@ Global
 		{6F5823CD-D1D7-4DF4-BE61-F07FD8134400}.Net45-Release|Any CPU.Build.0 = Net45-Release|Any CPU
 		{6F5823CD-D1D7-4DF4-BE61-F07FD8134400}.Portable-Debug|Any CPU.ActiveCfg = Portable-Debug|Any CPU
 		{6F5823CD-D1D7-4DF4-BE61-F07FD8134400}.Portable-Release|Any CPU.ActiveCfg = Portable-Release|Any CPU
-		{B419782D-CA57-40F9-9322-340A4E852E82}.Net40-Debug|Any CPU.ActiveCfg = Net40-Debug|Any CPU
-		{B419782D-CA57-40F9-9322-340A4E852E82}.Net40-Debug|Any CPU.Build.0 = Net40-Debug|Any CPU
-		{B419782D-CA57-40F9-9322-340A4E852E82}.Net40-Release|Any CPU.ActiveCfg = Net40-Release|Any CPU
-		{B419782D-CA57-40F9-9322-340A4E852E82}.Net40-Release|Any CPU.Build.0 = Net40-Release|Any CPU
-		{B419782D-CA57-40F9-9322-340A4E852E82}.Net45-Debug|Any CPU.ActiveCfg = Net45-Debug|Any CPU
-		{B419782D-CA57-40F9-9322-340A4E852E82}.Net45-Debug|Any CPU.Build.0 = Net45-Debug|Any CPU
-		{B419782D-CA57-40F9-9322-340A4E852E82}.Net45-Release|Any CPU.ActiveCfg = Net45-Release|Any CPU
-		{B419782D-CA57-40F9-9322-340A4E852E82}.Net45-Release|Any CPU.Build.0 = Net45-Release|Any CPU
+		{B419782D-CA57-40F9-9322-340A4E852E82}.Net40-Debug|Any CPU.ActiveCfg = Portable-Debug|Any CPU
```

Comment:
This change was intentional inasmuch as we have the following requirements:

- KeyVault.Core should only be built and packaged as Portable
- KeyVault and KeyVault.Extensions should only be Net40+BCL and Net45, Portable is not possible for these packages because of the crypto code that we use.

If there is better solution that allows a bland command line or VS build to work without selecting specific FxTargetList for each of our components, I'd be happy to adopt that solution.